cluster/pulumi/common/src/utils.ts

```diff
@@ -91,7 +91,18 @@ export const sequencerResources: { resources?: k8s.types.input.core.v1.ResourceR
           },
         },
       }
-    : {};
+    : {
+        resources: {
```

Comment:
I think this one we can drop, we changed the defaults in main already to have higher values than this.